README.md

```diff
@@ -9,12 +9,12 @@ Task:
 P.S
 `draw` - means print figure and all its' properties in the console using System.out.println()  
 1. We need to generate an array of figures with random parameters, number of objects in the array is also unknown.
-For this purpose create two more classes (`ColorSupplier` - for generating random color and `FigureSupplier` - for generating figure with random properties)
+For this purpose create two more classes (`ColourSupplier` - for generating random color and `FigureSupplier` - for generating figure with random properties)
 1. After generating the array, we need to display the entire list of objects that we have, for example:
 
 ```
- Figure: square, area: 25 sq. units, side length: 5 units, color: blue
- Figure: triangle, area: 12.5 sq. units, hypotenuse: 7.1 units, color: yellow
+ Shape: square, area: 25 sq. units, side length: 5 units, color: blue
+ Shape: triangle, area: 12.5 sq. units, hypotenuse: 7.1 units, color: yellow
 ```
```

Comment:
let's not change anything in task description - in green are the changes that you made - let's fix the changes